pyserini/search/lucene/ltr/__main__.py

```diff
@@ -119,17 +134,47 @@ def dev_data_loader(file, format, data, rerank, prebuilt, top=1000):
     return dev, dev_qrel
 
 
-def query_loader():
+def query_loader(topic):
     queries = {}
-    with open(f'{args.queries}/queries.dev.small.json') as f:
-        for line in f:
-            query = json.loads(line)
-            qid = query.pop('id')
-            query['analyzed'] = query['analyzed'].split(" ")
-            query['text'] = query['text_unlemm'].split(" ")
-            query['text_unlemm'] = query['text_unlemm'].split(" ")
-            query['text_bert_tok'] = query['text_bert_tok'].split(" ")
-            queries[qid] = query
+    if os.getcwd().endswith('ltr_msmarco'):
+        stopwords = read_stopwords('stopwords.txt', lower_case=True)
+    else:
+        stopwords = read_stopwords('./scripts/ltr_msmarco/stopwords.txt', lower_case=True)
```

Comment:
the code in the "module" can't depend on things in `scripts/`... because in the PyPI installation, `scripts/` does not exist.